### ISSUE TITLE
What's the "spatial ratio"?

### ISSUE THREAD

**Issue by Bonsen** [lianqing11/PyCDA#2]
I can't find it in yr paper.
And what's the "weighted_softmax" in train_pycda.py?

**Comment by lianqing11** [comment]
Hi Bonsen,
    "spatial ratio" is the spatial frequency distribution used in [CBST](https://arxiv.org/abs/1810.07911). 
    "Weighted_softmax" is the frequency distribution over a global image. It's referred to the curriculum loss in the paper.

**Comment by Xcrid** [comment]
Hi, 

Am i right to assume that this spatial ratio/spatial priors is what is named "local_array.npy" in your project ?

And why do you remove poly learning rate when you use spatial ratio ?

**Comment by lianqing11** [comment]
Hi Xcrid,

Yes, it is in "local_array.npy"

In the spatial priors, I have decreased the initial learning rate by 0.1 due to the the batch size issue, so I remove it during training.
